src/FunctionInfo.ts

```diff
@@ -13,13 +13,15 @@ export class FunctionInfo {
     [key: string]: rpc.IBindingInfo & { converter: (any) => rpc.ITypedData }
   };
   public httpOutputName: string;
+  public isHttp: boolean;
 
   constructor(metadata: rpc.IRpcFunctionMetadata) {
     this.name = <string>metadata.name;
     this.directory = <string>metadata.directory;
     this.bindings = {};
     this.outputBindings = {};
     this.httpOutputName = "";
+    this.isHttp = false;

```

Comment:
nit: consider a name with trigger in the name, like `hasHttpTrigger`. `isHttp` is a little vague. Given the context of `FunctionInfo`, `isHttp` is probably clear enough to mean it uses an HTTP trigger, so if you prefer the name, keep it.